x-pack/plugins/index_lifecycle_management/server/routes/api/policies/register_create_route.ts

```diff
@@ -51,10 +48,10 @@ export function registerCreateRoute({
     { path: addBasePath('/policies'), validate: { body: bodySchema } },
     license.guardApiRoute(async (context, request, response) => {
       const body = request.body as typeof bodySchema.type;
-      const { name, phases } = body;
+      const { name, ...rest } = body;
 
       try {
-        await createPolicy(context.core.elasticsearch.client.asCurrentUser, name, phases);
+        await createPolicy(context.core.elasticsearch.client.asCurrentUser, name, { ...rest });
```

Comment:
nit: I think we can directly pass `rest`, no need to deconstruct it a second time 😊